wiki/Main_Page/en.md

```diff
@@ -3,9 +3,11 @@ layout: "main_page"
 ---
 
 <div class="wiki-main-page__blurb">
+
```

Comment:
the other markdown-in-html parts use 3 newlines at the start, is that needed here?

Reply:
Nah, you could just remove all those newlines.

Reply:
i mean those under the blurb ofc

Reply:
I'm asking about something like line 13-16 cuz it has 3 newlines there but you only use 2 here

Reply:
How I understand it is that it didn't matter here since it was separating two html elements. You would have to ensure there's enough newlines here if there were any markdown elements between them.